building/build_from_commandline.adoc

```diff
@@ -0,0 +1,30 @@
+#  Building locally via commandline
+
+First, fetch the sources of the project you want to build from Github. For details, see link:configuring_git_workflow.adoc#Sources[Configuring Git Workflow - Sources].
+
+Then, switch into the directory where you checked out the sources, and run this:
+
+
+[source,bash]
+----
+cd ~/my-github-stuff/jbosstools-base
+
+# build everything and run all tests
```

Comment:
-fae is not recommended when building locally for development. Since you then will have to wait to the end to get the errors. -fae is more for CI builds or when you are trying to build a multimodule project updatesite forcefully ignoring errors.